x-pack/platform/plugins/shared/lens/public/editor_frame_service/error_helper.tsx

```diff
@@ -10,6 +10,21 @@ import { renderSearchError } from '@kbn/search-errors';
 import React from 'react';
 import type { UserMessage } from '@kbn/lens-common';
 
+export const ignoredBlockingMessagesByID = new Set<string>(['The expression was aborted.']);
+
+function extractUniqueId(message: string | { short: string; long: React.ReactNode }): string {
+  return typeof message === 'string' ? message : message.short;
+}
+
+export function isIgnorableBlockingMessage(error: ExpressionRenderError | null): boolean {
+  const message = error?.original?.message ?? error?.message;
+  if (!message) {
+    return false;
+  }
+  const uniqueId = extractUniqueId(message);
+  return ignoredBlockingMessagesByID.has(uniqueId);
+}
```

Comment:
Instead of relying on the `message` can't we revisit the `ExpressionRenderError` error adding a property that describe if this can be ignored or not? or giving a Tag or an ID that doesn't rely on the message itself?
We can also consider changing this from a general name `AbortError` to `ExpressionAbortError' to make it clear and less error prone

Reply:
> Instead of relying on the message can't we revisit the ExpressionRenderError error adding a property that describe if this can be ignored or not?

This would impose a Lens logic to the entire Kibana expression system: the fact that Lens wants to skip this specific error should not influence other tools, unless we all agree in Kibana to ignore this specific error type IMHO.